instrumentation/apache-elasticjob-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apacheelasticjob/v3_0/ElasticJobTest.java

```diff
@@ -150,7 +154,7 @@ void testSimpleJob() {
             regCenter,
             job,
             JobConfiguration.newBuilder("simpleElasticJob", 2)
-                .cron("0/5 * * * * ?")
+                .cron("0/30 * * * * ?")
```

Comment:
this was causing some headaches while i was refactoring, due to the small interval there were more spans created than expected